files/zh-cn/mdn/writing_guidelines/howto/markdown_in_mdn/index.md

```diff
@@ -411,19 +414,19 @@ const greeting = "I'm a bad example";
 
 ```md
 | Heading 1 | Heading 2 | Heading 3 |
-|-----------|-----------|-----------|
+| --------- | --------- | --------- |
 | cell 1    | cell 2    | cell 3    |
 | cell 4    | cell 5    | cell 6    |
 ```
 
 而不是这种风格：
 
 ```md
-Heading 1 | Heading 2 | Heading 3
- --- | --- | ---
+| Heading 1 | Heading 2 | Heading 3 |
+| --------- | --------- | --------- |
```

Comment:
This is a bad-example here, we should add `-nolint` :)

I've created a PR to revert the changes in en-US: mdn/content#27978

Reply:
I'm down to add `-nolint` here for now until the upstream version's updated and we resync with upstream!